pkg/registry/core/service/storage/rest_test.go

```diff
@@ -622,48 +622,210 @@ func TestServiceRegistryUpdate(t *testing.T) {
 }
 
 func TestServiceRegistryUpdateUnspecifiedAllocations(t *testing.T) {
+	type proof func(t *testing.T, s *api.Service)
+	prove := func(proofs ...proof) []proof {
+		return proofs
```

Comment:
Low pri

 on all proves ( don't think we should fix but we should ack it). `idx` is never tested against `len(<the thing we want to test>)`. This could yield into undesirable error, specially when i think these should be part of `make.go`

Reply:
The good news is that they will panic if wrong.

I agree finding a way to re-use these will be nice - as a followup to the mega PR :)